### DISCUSSION TITLE
Error during column update [Jul 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6122] [Discord Threads]
> 🙋‍♂️ **semighoti**  
## Hi guys, I need help with this issue. Whenever i choose a value from last column, it shows up the error like the picture, but still save the value  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1119317571016077403/Screenshot_from_2023-06-16_10-26-18.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Help us with relevant details as a Git issue.    
#### https://github.com/nocodb/nocodb/issues/new/choose  
#  
> 👨‍🦲 **dstala**  
#### I was not able to reproduce locally  
#  
> 🙋‍♂️ **monjab**  
#### i have a save problem. pls tell me this is fix ?  
#  
> 👨‍🦲 **dstala**  
#### <@312209662260936716> Can you share more details about your setup? You can copy project-info from project menu & paste here  
#  
> 🙋‍♂️ **monjab**  
#### Node: **v16.19.1**   
#### Arch: **arm64**   
#### Platform: **linux**   
#### Docker: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.107.5** this my project info. thx for respond!  
#  
> 🙋‍♂️ **monjab**  
#### project with problems it import form airtable.  
#  
> 🙋‍♂️ **monjab**  
#### Maybe it's because I deleted the column import from the airtable...  
#  
> 🙋‍♂️ **monjab**  
#### is it related?  
#  
> 🙋‍♂️ **semighoti**  
#### I got it fixed, i have not encountered this error anymore  
#  
> 👨‍🦲 **dstala**  
#### <@312209662260936716> Try upgrading to latest version; if you still face this issue - ping back here  
#  
> 🙋‍♂️ **monjab**  
#### thank!. i will try now.  
#  
> 🙋‍♂️ **monjab**  
#### <@857998597474484295> i still have a problem it same error.  
![image0](https://cdn.discordapp.com/attachments/1119317571137720431/1122093241974669372/image.png)  
#  
> 🙋‍♂️ **semighoti**  
#### Hey <@312209662260936716> are u using external database?  
#  
> 🙋‍♂️ **monjab**  
#### i think it a internal database.but my data it import form airtable.  
![image0](https://cdn.discordapp.com/attachments/1119317571137720431/1122497072478818394/image.png)  
#  
> 🙋‍♂️ **semighoti**  
#### I use postgres for my data. This is what i did. I backup everything, then delete the noco_db (not postgres database) then docker down and docker up. After that, i sync postgres to nocodb again, all errors are gone  
#  
> 🙋‍♂️ **semighoti**  
#### But you have to setup grid view again. I guess because I delete data from database level, so some of the data have not been deleted on nocodb, so the error persists  
#  
> 🙋‍♂️ **monjab**  
#### i found another way...  
#  
> 🙋‍♂️ **monjab**  
#### it easy but.i don't know it have a fuction!  
#  
> 🙋‍♂️ **monjab**  
#### we can clear cache to sync database!  
#  
> 🙋‍♂️ **monjab**  
####   
![image0](https://cdn.discordapp.com/attachments/1119317571137720431/1122746864362594414/image.png)  
#  
> 🙋‍♂️ **monjab**  
#### yes this is same my problem!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1119317571137720431)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
